avm/res/sql/server/audit-setting/main.bicep

```diff
@@ -52,20 +52,37 @@ var primaryUserAssignedIdentityPrincipalId = filter(
   identity => identity.key == server.properties.primaryUserAssignedIdentityId
 )[0].value.principalId
 
-module storageAccount_sbdc_rbac 'modules/nested_storageRoleAssignment.bicep' = if (isManagedIdentityInUse && !empty(storageAccountResourceId)) {
+// If storage account is in a different resource group
+module storageAccount_sbdc_rbac_rg 'modules/nested_storageRoleAssignment.bicep' = if (isManagedIdentityInUse && !empty(storageAccountResourceId) && split(
```

Comment:
I take it the conditional scope did not work? Hence the split?
Saw some talks about it in [this](https://github.com/Azure/bicep/issues/15517#issuecomment-2906505661) issue